src/sascompoundheap.cpp

```diff
@@ -1669,3 +1673,4 @@ SPHCompoundPCQNearAlloc (void *nearObj)
     }
   return newHeap;
 }
+
```

Comment:
The changes look good in general except few extra  new empty lines added. The commit message did not appear. Could you please add that as a comment which I will use while merging the changes.

Reply:
[git-log-20180115.txt](https://github.com/sphde/sphde/files/1636614/git-log-20180115.txt)
[git-log-20180115b.txt](https://github.com/sphde/sphde/files/1636615/git-log-20180115b.txt)

Reply:
Looks like I have core.editor set:

sjmunroe@SJM-ThinkPad-W520:~/work/github/forks/sphde$ git config --global core.editor
gedit
sjmunroe@SJM-ThinkPad-W520:~/work/github/forks/sphde$ git config core.editor
gedit